Purchases/Classes/Public/RCPurchases.m

```diff
@@ -41,16 +41,60 @@ @interface RCPurchases () <RCStoreKitWrapperDelegate>
 
 @implementation RCPurchases
 
+static RCPurchases *_singleton = nil;
+
+@synthesize delegate=_delegate;
```

Comment:
Can we kill this?

Reply:
we are overriding the setter so we need to keep it